internal/pkg/template/template.go

```diff
@@ -53,18 +57,22 @@ func New(fs filesystem.Fs, manifest *Manifest, inputs *Inputs, d dependencies) *
 	}
 }
 
-func (p *Template) Fs() filesystem.Fs {
-	return p.fs
+func (t *Template) Fs() filesystem.Fs {
+	return t.fs
+}
+
+func (t *Template) Manifest() manifest.Manifest {
+	return t.manifest
 }
 
-func (p *Template) Manifest() manifest.Manifest {
-	return p.manifest
+func (t *Template) Inputs() *Inputs {
+	return t.inputs
 }
 
-func (p *Template) Inputs() *Inputs {
-	return p.inputs
+func (t *Template) Ctx() context.Context {
+	return context.WithValue(t.dependencies.Ctx(), validator.DisableRequiredInProjectKey, true)
```

Comment:
V template state musime vypnut validaciu `required_in_project` (uz sme to spravili pre template manifest).
Ostatne zmeny su uz iba o tom, dostat `context` do validator.